src/main/markdown/makinggwtbetter.md

```diff
@@ -512,17 +512,17 @@ necessary.
     <th>Tolerable</th>
   </tr>
   <tr>
-    <td>`Foo<T>`</td>
-    <td>`Foo<FooClientType>`</td>
+    <td>`Foo&lt;T&gt;`</td>
```

Comment:
If `<` doesn't display properly, that means backticks aren't working properly, and they should be replaced with `<code>` too.

Either that or use markdown for the table rather than HTML (I'd prefer that, easier to read).